carbon-design-system/carbon-addons-iot-react
# fix(DashboardEditor): update styling to match design

Internal issues: https://github.ibm.com/wiotp/monitoring-dashboard/issues/1748 and https://github.ibm.com/wiotp/monitoring-dashboard/issues/1746

**Summary**

- Current DashboardEditor styling does not match design specs as seen in the above issues

**Change List (commits, features, bugs, etc)**

- Added new classname to the Data item list in the CardEditor to get specificity to force the borders to go full width
- Updated breakpoints to be Small, Medium, Large
- Updated breakpoint messages and functions to match new accepted sizes
- Updated breakpoint outline to use the lower width range
- Created new common component called ContentFormItemTitle as we were reusing the same code in 3 different locations: DataSeriesFormContent, TableFormContent, and ImageFormContent
- Added `dataSeriesItemLinks` prop to DashboardEditor to render links within tooltips of the data series sections for each card type that are card type-specific

**Acceptance Test (how to verify the PR)**

- Go to /?path=/story/watson-iot-experimental-dashboardeditor--with-breakpoint-switcher and check that the above changes match the specs seen in the issues above
- Select each card type and ensure a tooltip with description is rendering correctly
- Go to card editor - tooltip link story to ensure the link is rendering correctly
